buildlib/azure-pipelines-pr.yml

```diff
@@ -23,6 +23,8 @@ pr:
     - buildlib/azure-pipelines-perf.yml
     - buildlib/tools/perf_results.py
     - buildlib/tools/perf-common.yml
+    - buildlib/azure-pipelines-release.yml
+    - buildlib/jucx/*
```

Comment:
aren't we building jucx also for PR?

Reply:
jucx-test.yml is used in PR indeed, changing.